docs/technology/transaction-lifecycle.mdx

```diff
@@ -24,7 +24,7 @@ Transactions on Linea proceed through the following steps:
 ## Step 1: Submission
 
 This is where every transaction originates: at the wallet level, a user signs a transaction and 
-broadcasts it to Linea. From here, transactions head straight to the **mempool**, similarly to 
+broadcasts it to Linea. From here, transactions head straight to the **mempool**, similar to 
```

Comment:
```suggestion
broadcasts it to Linea. From here, transactions head straight to the **mempool**, similarly to 
```

I think either is valid here -- the adverb `similarly` feels appropriate as it's describing a similarity with the action of transactions heading to the mempool.